tests/publiccloud/instance_overview.pm

```diff
@@ -26,6 +26,7 @@ sub run {
 
     assert_script_run("hostname -f");
     assert_script_run("uname -a");
+    script_run("rpm -qi kernel-default");
```

Comment:
What is your intention? Get info's about `kernel-default` but don't care if it is `kernel-azure`? I wouldn't add a none `assert_script_run()` logic.

What about:
```
assert_script_run(q(rpm -qa 'kernel*' --qf '%{NAME}\n' | sort | uniq | xargs rpm -qi));
```

Reply:
nice I like that ! ok I will change it to your version

Reply:
> And maybe add a `record_info()` to have that info longer then the assets exists...

yup, that's why I do in the block tests. After all this is meant as a helper to support investigation in case of failing test.

Reply:
BTW we use similar logging. I wonder if some code like this should be added into lib/upload_system_log.pm, to prevent everybody reimplementing a wheel. But I like that idea of `record_info()` (the persistency).